daemon/system-info.sh

```diff
@@ -8,111 +8,182 @@ KERNEL_VERSION="$(uname -r)"
 ARCHITECTURE="$(uname -m)"
 
 # -------------------------------------------------------------------------------------------------
-# detect the operating system
+# detect the virtualization
 
-OS_DETECTION="unknown"
-NAME="unknown"
-VERSION="unknown"
-VERSION_ID="unknown"
-ID="unknown"
-ID_LIKE="unknown"
+if [ -z ${VIRTUALIZATION} ]; then
```

Comment:
Please add double quote around "${VIRTUALIZATION}" to remove the following warning from `shellcheck` and, of course, to avoid problems with variable content:

```
In daemon/system-info.sh line 13:
if [ -z ${VIRTUALIZATION} ]; then
        ^---------------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
Fixed.